src/extractor/edge_based_graph_factory.cpp

```diff
@@ -138,7 +138,7 @@ void EdgeBasedGraphFactory::InsertEdgeBasedNode(const NodeID node_u, const NodeI
     NodeID current_edge_source_coordinate_id = node_u;
 
     // traverse arrays from start and end respectively
-    for (const auto i : util::irange(0UL, geometry_size))
+    for (const auto i : util::irange(std::size_t{ 0 }, geometry_size))
```

Comment:
These are the error messages: https://github.com/Project-OSRM/osrm-backend/pull/2118#issuecomment-199802077
I think VS is not able to deduct which type `0UL` is.